src/components/OnboardingList.js

```diff
@@ -21,6 +22,7 @@ function OnboardingCheckListItem({ children, actions, state, ...props }) {
       className={classNames(
         "nfd-p-[2px]",
         "nfd-m-0 nfd-border-b nfd-border-line last:nfd-border-b-0",
+        state.className,
```

Comment:
Can you pass `className` as props

Reply:
I'm passing classname as prop which is being destructed as state from props, these values are coming from onboarding.config.js